lib/PHPExif/Mapper/Exiftool.php

```diff
@@ -394,7 +396,7 @@ protected function extractGPSCoordinates(string $coordinates) : float|false
         if (is_numeric($coordinates) === true || $this->numeric === true) {
             return ((float) $coordinates);
         } else {
-            if (!preg_match('!^([0-9.]+) deg ([0-9.]+)\' ([0-9.]+)"!', $coordinates, $matches)) {
+            if (0 === preg_match('!^([0-9.]+) deg ([0-9.]+)\' ([0-9.]+)"!', $coordinates, $matches)) {
```

Comment:
I don't know how far back you go in programming, but "back in the day" some advocated for that style in C, where `if (a = 0)` and `if (a == 0)` are both valid constructs but the first one is usually a typo. So if you reverse the order (`if (0 = a)` and `if (0 == a)`) the first one becomes invalid (since a constant can't be an lvalue). Nowadays compilers routinely warn about `if (a = 0)` and one needs another pair of brackets to avoid that message, so it's not really an issue anymore.